src/xml2csv.py

```diff
@@ -1,90 +1,81 @@
-import csv
-import gc
 import json
-import os
 import sys
+from csv import DictWriter
 from datetime import datetime
-from tempfile import TemporaryDirectory
+from io import StringIO
 
-import numpy as np
 from bs4 import BeautifulSoup
```

Comment:
Yes. I mean, to parse the HTML table BeautifulSoup still useful here and at `src/translation_table.py` — but I think we can use *here* an import and do the HTML parsing in `src/translation_table.py`. Let's merge both (as they are on separate branches) and them I look for duplicates and try to eliminate them. What do you think?